core/domain/suggestion_registry.py

```diff
@@ -535,7 +535,7 @@ def pre_update_validate(self, change):
             raise utils.ValidationError(
                 'The new change skill_id must be equal to %s' %
                 self.change.skill_id)
-        if self.change.question_domain == change.question_dict:
+        if self.change.question_dict == change.question_dict:
```

Comment:
Wow didn't expect that bug! Great catch.

Reply:
Thanks :)